### PR TITLE
Tickets/preops 4604

### PR DESCRIPTION
**Pull request by cristobalpio**

### CODE REVIEW

File: src/lsst/cst/__init__.py [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
 
+from .mpl_utils import remove_figure
+
 __version__: str
 """The version string of lsst
 (PEP 440 / SemVer compatible).
 """
 
+
+__all__ = ["__version__", "remove_figure"]
```
Comment (mfisherlevine) [inline src/lsst/cst/__init__.py:13]: I think `__all__` is usually a tuple for immutability.

**Review by mfisherlevine — APPROVED** [review]
Look good to me, just one minor, optional comment.

File: src/lsst/cst/__init__.py (hunk 1 above)
Comment (timj) [reply src/lsst/cst/__init__.py:13]: Except all the Python docs use a list for their examples...

Comment (mfisherlevine) [reply src/lsst/cst/__init__.py:13]: Like I said, it's an optional suggestion 🙂

Comment (mfisherlevine) [reply src/lsst/cst/__init__.py:13]: PEP8 (and Tim/docs) say list, so probably keep it as a list 👍

**Pull request merged**